### PR TITLE
AFC-1476b (AFC-1544) auth_user_rights_wizard Action Popup for Checkboxes in the Form

### PR DESCRIPTION
**Pull request by iammiracle**

### CODE REVIEW

File: auth_user_rights_wizard/views/hr_employee_view.xml [hunk 1]
```diff
@@ -24,35 +24,35 @@
...
+<!--            <field name="type">ir.actions.act_window</field>-->
+<!--            <field name="res_model">hr.employee.user.right</field>-->
+<!--            <field name="domain">[('id','in',active_ids)]</field>-->
+<!--            <field name="view_type">form</field>-->
+<!--            <field name="view_mode">form</field>-->
+<!--            <field name="target">new</field>-->
+<!--            <field name="view_id" ref="view_access_right_wizard"/>-->
+<!--            <field name="binding_model_id" ref="hr.model_hr_employee"/>-->
+<!--            <field name="groups_id" eval="[(4, ref('base_user_groups_dafa.group_dafa_superadmin'))]"/>-->
+<!--        </record>-->
```
Comment (arturchilangwa-vertel) [inline auth_user_rights_wizard/views/hr_employee_view.xml]: Please state why you want to keep this code, with a code comment. Thanks.

File: auth_user_rights_wizard/views/hr_employee_view.xml [hunk 2]
```diff
@@ -24,35 +24,35 @@
...
+<!--                        <field name="employee_id" invisible="1"/>-->
+<!--                        <field name="group_id" required="1" widget="many2many_tags" options="{'no_create': True, 'no_open': True}"/>-->
+<!--                    </group>-->
+<!--                    <footer>-->
+<!--                        <button string="Assign" type="object" name="action_assign_rights" class="btn-primary" />-->
+<!--                        <button string="Cancel" special="cancel" class="btn-secondary"/>-->
+<!--                    </footer>-->
+<!--                </form>-->
+<!--            </field>-->
+<!--        </record>-->
```
Comment (arturchilangwa-vertel) [inline auth_user_rights_wizard/views/hr_employee_view.xml]: Please state why you want to keep this code, with a code comment. Thanks.

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]

File: auth_user_rights_wizard/views/hr_employee_view.xml (hunk 1 above)
Comment (arturchilangwa-vertel) [reply auth_user_rights_wizard/views/hr_employee_view.xml]: @iammiracle

File: auth_user_rights_wizard/views/hr_employee_view.xml (hunk 2 above)
Comment (iammiracle) [reply auth_user_rights_wizard/views/hr_employee_view.xml]: Nothing in particular, I don't know if we will need to implement it again, but I can remove it

**Review by daerio — APPROVED** [review]

**Pull request merged**